src/tests/unit/tests/common/get-card-selection-view-data.test.ts

```diff
@@ -16,6 +17,8 @@ describe('getCardSelectionStoreviewData', () => {
     let initialUnifiedScanResultState: UnifiedScanResultStoreData;
     let isResultHighlightUnavailable: IMock<IsResultHighlightUnavailable>;
 
+    const resultsFilter: ResultsFilter = res => res.uid == 'sampleUid3';
```

Comment:
This filter is very tightly coupled to the specific expectation on L101-103; I think it would be a little more readable to move it into the test case it's coupled to.